frontend-zs/src/components/stories/StoryPage.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Media, Container } from 'react-bootstrap'
+import Axios from 'axios'
+import { Link, useParams } from 'react-router-dom'
+import { TiArrowBack } from "react-icons/ti"
+import StoryDetails from './StoryDetails'
+import StoryPagination from './StoryPagination'
+import StoryQuestionLike from './StoryQuestionLike'
+import { authenticationService } from '../../services/authentication.service';
+import './Story.scss'
+
+function goBack() {
+    return (
+        <Button className="btn-back mb-3" size="sm" active>
+            <Link to="/stories" >
+                <Media>
+                    <TiArrowBack size={20} />
+                    <Media.Body>Back</Media.Body>
+                </Media>
+            </Link>
+        </Button>
+    )
+}
+
+function StoryText(displayStory) {
+    return (
+        <div className="p-4 new-line" style={{ fontSize: 19 }} >
+            {displayStory}
+        </div>
+    )
+}
+
+export default function StoryPage(props) {
+    const params = useParams();
+    const [story, setStory] = useState({
+        title: '',
+        author: '',
+        authorImage: '',
+        storyPages: [],
+        categories: [],
+        numberLikes: []
+    })
+    const [displayStory, setDisplayStory] = useState('')
+    const [currentUserId, setCurrentUserId] = useState('')
+
+    useEffect(() => {
+        Axios.get(process.env.REACT_APP_HOST + ':' + process.env.REACT_APP_PORT
+            + '/api/stories/' + props.computedMatch.params.id)
+            .then(response => {
+                setStory(response.data)
+            })
+            .catch(function (error) {
+                console.log(error.message)
+            });
+        const currentUser = authenticationService.currentUserValue;
+        setCurrentUserId(currentUser._id)
+    }, [])
```

Comment:
This triggers a warning.. Would adding ``props.computedMatch.params.id`` to the dependencies work the same?

```suggestion
    }, [props.computedMatch.params.id])
```